packages/theme-default/src/components/Card/index.tsx

```diff
@@ -0,0 +1,36 @@
+interface CardProps {
+  /**
+   * The content to display inside the card. Can be a string or React nodes.
+   */
+  children: React.ReactNode;
+  /**
+   * The title of the card.
+   */
+  title?: React.ReactNode;
+  /**
+   * The icon of the card.
+   */
+  icon?: React.ReactNode;
+  /**
+   * The style of the card.
+   */
+  style?: React.CSSProperties;
+  /**
+   * The content to display inside the card.
+   */
+  text?: string;
```

Comment:
can use only one field named `content?: React.ReactNode`

Reply:
Does it mean to change `text` to `content` and remove `children`?

Reply:
seems both `text` and `children` represents the content to display inside the card, we can only use one field.

Reply:
Ok